src/static/orderform.php

```diff
@@ -1,3 +1,5 @@
+
+
```

Comment:
since we have consolidated all functionality into order.php this page can be deleted